api/integrations/webhook/serializers.py

```diff
@@ -0,0 +1,48 @@
+import typing
+
+from django.db.models import Q
+from rest_framework import serializers
+
+from features.serializers import FeatureStateSerializerFull
+from segments.models import Segment
+
+from .models import WebhookConfiguration
+
+
+class WebhookConfigurationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = WebhookConfiguration
+        fields = ("id", "url", "secret")
+
+
+class SegmentSerializer(serializers.Serializer):
+    member = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Segment
+        fields = ("id", "name", "member")
+
+    def get_member(self, obj):
+        return obj.does_identity_match(identity=self.context.get("identity"))
+
+
+class IntegrationFeatureStateSerializer(FeatureStateSerializerFull):
+    def to_representation(self, instance):
+        ret = super().to_representation(instance)
```

Comment:
What is `ret`?

Reply:
Return value(yep, I see it's not very intuitive, should we change it?https://www.django-rest-framework.org/community/3.0-announcement/), this is sort of a convention in drf:
```
def to_representation(self, instance):
    ret = super(UserSerializer, self).to_representation(instance)
    ret['username'] = ret['username'].lower()
    return ret

class BaseModelSerializer(ModelSerializer):
    """
    A custom ModelSerializer class that preserves 2.x style `transform_<field_name>` behavior.
    """
    def to_representation(self, instance):
        ret = super(BaseModelSerializer, self).to_representation(instance)
        for key, value in ret.items():
            method = getattr(self, 'transform_' + key, None)
            if method is not None:
                ret[key] = method(value)
        return ret
```
src: https://www.django-rest-framework.org/community/3.0-announcement/

Reply:
(I've approved it regardless)